ansible-role-rocky-requirements.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014-2017, Rackspace US, Inc.
```

Comment:
In order to match RPC_PRODUCT_RELEASE this content should go into ``ansible-role-master-requirements.yml`` instead. The ``ansible-role-rocky-requirements.yml`` file will only be created once the ``rocky`` branch is cut.